src/gleam/string.gleam

```diff
@@ -803,3 +803,22 @@ pub fn capitalise(s: String) -> String {
     _ -> ""
   }
 }
+
+if javascript {
+  pub fn inspect(value: a) -> String {
+    do_inspect(value)
+  }
+
+  external fn do_inspect(value: a) -> String =
+    "../gleam.mjs" "inspect"
+}
+
+if erlang {
+  pub fn inspect(value: a) -> String {
+    do_inspect(value)
+    |> string_builder.to_string
```

Comment:
Can we put this in the `inspect` function and make the `do_inspect` function return `StringBuilder` please? To remove this extra wrapper function

Reply:
I don't understand why this works, but it does, tests run: `"../gleam.mjs" "inspect"` to my understanding returns a gleam `Map` not a `StringBuilder`.